frontend/tests/utils/helpers.ts

```diff
@@ -9,21 +15,30 @@ export async function goToPage(url: string = TEST_URL) {
 
 export async function login() {
   goToPage();
+  expect(process.env.TEST_ACCOUNT_PASS).toBeDefined();
   try {
     await expect(page).toHaveSelector(getText("My Collections"), {
       timeout: TIMEOUT_MS,
     });
   } catch (error) {
     const url = page.url();
-    await page.click(getText("Log In"));
-
-    await page.fill('[name="email"]', TEST_EMAIL);
-    // await page.fill('[name="password"]', password);
-
-    await Promise.all([
-      page.waitForNavigation({ waitUntil: "networkidle" }),
-      page.click('[name="submit"]'),
-    ]);
+    if (TEST_ENV === "happy") {
```

Comment:
I think we can simplify this with Playwright's new selector too!

```ts
await page.fill(`[name="email"], [name="Username"]`, TEST_EMAIL);
await page.fill(`[name="password"], [name="Password"]`, TEST_PASSWORD);
```

Found this yesterday 😆 

https://playwright.dev/docs/selectors#selecting-elements-matching-one-of-the-conditions

Reply:
Wicked, I'll do that now, same with the submit button!